images/virtualization-artifact/pkg/controller/cvi/cvi_webhook.go

```diff
@@ -41,8 +43,15 @@ func NewValidator(logger *log.Logger) *Validator {
 }
 
 func (v *Validator) ValidateCreate(_ context.Context, obj runtime.Object) (admission.Warnings, error) {
-	err := fmt.Errorf("misconfigured webhook rules: create operation not implemented")
-	v.logger.Error("Ensure the correctness of ValidatingWebhookConfiguration", "err", err)
+	cvi, ok := obj.(*virtv2.ClusterVirtualImage)
+	if !ok {
+		return nil, fmt.Errorf("expected a new ClusterVirtualImage but got a %T", obj)
+	}
+
+	if strings.Contains(cvi.ObjectMeta.Name, ".") {
+		return nil, errors.New("cluster virtual image name cannot contain '.'")
```

Comment:
```suggestion
		return nil, errors.New("VlusterVirtualImage name is invalid: '.' is forbidden, allowed name symbols are [0-9a-zA-Z-]")
```

Please, correct me if symbols set is different.

Also, change messages for VirtualImage.